Makefile

```diff
@@ -25,3 +25,7 @@ tests:
 .PHONY: benchmarks
 benchmarks:
 	make -C tests benchmarks
+
+.PHONY: system-tests
```

Comment:
Go ahead and name this `google-cloud-system-tests`, as we may want to add system tests for other packages in the future?